### PR TITLE
Use libfmt for printing

### PR DESCRIPTION
**Pull request by agutenkunst**
For more infos on libfmt see https://github.com/fmtlib/fmt

### CODE REVIEW

File: include/psen_scan_v2/logging.h [hunk 1]
```diff
@@ -18,19 +18,18 @@
...
 #include <sstream>
+#include <fmt/core.h>
 
-#define PSENSCAN_LOG(name, file, line, level, args)                                                                    \
+#define PSENSCAN_LOG(name, file, line, level, ...)                                                                     \
   {                                                                                                                    \
-    std::stringstream ss;                                                                                              \
-    ss << name << ": " << args;                                                                                        \
-    console_bridge::getOutputHandler()->log(ss.str(), level, file, line);                                              \
+    console_bridge::getOutputHandler()->log(fmt::format(__VA_ARGS__), level, file, line);                              \
```
Comment (martiniil) [inline include/psen_scan_v2/logging.h:25]: I am missing the name argument.

**Review by martiniil — CHANGES_REQUESTED** [review]
I like it.

**Comment by hslusarek** [comment]
Looks good. Let's give it a try.

File: include/psen_scan_v2/logging.h (hunk 1 above)
Comment (agutenkunst) [reply include/psen_scan_v2/logging.h]: Yep is missed this. Fixed.

**Pull request merged**